Three MSLG tests failing

See npm run coveralls task in build 135 at https://fuselabs.visualstudio.com/SDK_v4/_build/results?buildId=57963&view=logs

 13 passing (29s)
  3 failing

  1) The mslg cli tool
       should output correctly
         should collate templates successfully:
     Error: ENOENT: no such file or directory, unlink '/Users/vsts/agent/2.150.3/work/1/s/packages/MSLG/finalResult_mslg.lg'
      at Object.unlinkSync (fs.js:951:3)
      at exec (test/mslg.cli.test.suite.js:118:24)
      at ChildProcess.exithandler (child_process.js:285:7)
      at maybeClose (internal/child_process.js:970:16)
      at Socket.stream.socket.on (internal/child_process.js:389:11)
      at Pipe._handle.close (net.js:597:12)

  2) The mslg cli tool
       should output correctly
         should collate templates from subfolder successfully:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at exec (test/mslg.cli.test.suite.js:131:28)
      at ChildProcess.exithandler (child_process.js:285:7)
      at maybeClose (internal/child_process.js:970:16)
      at Socket.stream.socket.on (internal/child_process.js:389:11)
      at Pipe._handle.close (net.js:597:12)

  3) The mslg cli tool
       should output correctly
         should write collated lg file to specific folder:
     Error: ENOENT: no such file or directory, unlink '/Users/vsts/agent/2.150.3/work/1/s/packages/MSLG/examples/validExamples/subValidExamples/finalResult_mslg.lg'
      at Object.unlinkSync (fs.js:951:3)
      at exec (test/mslg.cli.test.suite.js:143:24)
      at ChildProcess.exithandler (child_process.js:285:7)
      at maybeClose (internal/child_process.js:970:16)
      at Socket.stream.socket.on (internal/child_process.js:389:11)
      at Pipe._handle.close (net.js:597:12)

--- a/packages/MSLG/test/mslg.cli.test.suite.js
+++ b/packages/MSLG/test/mslg.cli.test.suite.js
@@ -115,7 +115,6 @@ describe('The mslg cli tool', function () {
             let filePath = resolvePath('examples/validExamples');
             exec(`node ${mslg} parse -l ${filePath} --out finalResult -c`, (error, stdout, stderr) => {
                 try {
-                    // fs.unlinkSync(generatedFilePath + '/finalResult_mslg.lg');
                     assert.equal(stdout.includes('Collated lg file is generated here'), true);
                     done();
                 } catch (err) {
@@ -140,7 +139,6 @@ describe('The mslg cli tool', function () {
             let filePath = resolvePath('examples/validExamples/simple.lg');
             exec(`node ${mslg} parse --in ${filePath} --out finalResult -o examples/validExamples/subValidExamples`, (error, stdout, stderr) => {
                 try {
-                    // fs.unlinkSync(generatedFilePath + '/finalResult_mslg.lg');
                     assert.equal(stdout.includes('Collated lg file is generated here'), true);
                     done();
                 } catch (err) {

--- a/packages/MSLG/test/mslg.cli.test.suite.js
+++ b/packages/MSLG/test/mslg.cli.test.suite.js
@@ -113,10 +113,9 @@ describe('The mslg cli tool', function () {
     describe('should output correctly', function () {
         it('should collate templates successfully', function (done) {
             let filePath = resolvePath('examples/validExamples');
-            let generatedFilePath = resolvePath('.');
             exec(`node ${mslg} parse -l ${filePath} --out finalResult -c`, (error, stdout, stderr) => {
                 try {
-                    fs.unlinkSync(generatedFilePath + '/finalResult_mslg.lg');
+                    // fs.unlinkSync(generatedFilePath + '/finalResult_mslg.lg');
                     assert.equal(stdout.includes('Collated lg file is generated here'), true);
                     done();
                 } catch (err) {
@@ -139,10 +138,9 @@ describe('The mslg cli tool', function () {
 
         it('should write collated lg file to specific folder', function (done) {
             let filePath = resolvePath('examples/validExamples/simple.lg');
-            let generatedFilePath = resolvePath('examples/validExamples/subValidExamples');
             exec(`node ${mslg} parse --in ${filePath} --out finalResult -o examples/validExamples/subValidExamples`, (error, stdout, stderr) => {
                 try {
-                    fs.unlinkSync(generatedFilePath + '/finalResult_mslg.lg');
+                    // fs.unlinkSync(generatedFilePath + '/finalResult_mslg.lg');
                     assert.equal(stdout.includes('Collated lg file is generated here'), true);
                     done();
                 } catch (err) {

--- a/packages/MSLG/test/mslg.cli.test.suite.js
+++ b/packages/MSLG/test/mslg.cli.test.suite.js
@@ -113,9 +113,10 @@ describe('The mslg cli tool', function () {
     describe('should output correctly', function () {
         it('should collate templates successfully', function (done) {
             let filePath = resolvePath('examples/validExamples');
+            let generatedFilePath = resolvePath('.');
             exec(`node ${mslg} parse -l ${filePath} --out finalResult -c`, (error, stdout, stderr) => {
                 try {
-                    fs.unlinkSync(resolvePath('finalResult_mslg.lg'));
+                    fs.unlinkSync(generatedFilePath + '/finalResult_mslg.lg');
                     assert.equal(stdout.includes('Collated lg file is generated here'), true);
                     done();
                 } catch (err) {
@@ -126,7 +127,7 @@ describe('The mslg cli tool', function () {
 
         it('should collate templates from subfolder successfully', function (done) {
             let filePath = resolvePath('examples/validExamples');
-            exec(`node ${mslg} parse -l ${filePath} -s --out finalResult -c --stdout`, (error, stdout, stderr) => {
+            exec(`node ${mslg} parse -l ${filePath} -s -c --stdout`, (error, stdout, stderr) => {
                 try {
                     assert.equal(stdout.includes('# Byebye'), true);
                     done();
@@ -138,9 +139,10 @@ describe('The mslg cli tool', function () {
 
         it('should write collated lg file to specific folder', function (done) {
             let filePath = resolvePath('examples/validExamples/simple.lg');
+            let generatedFilePath = resolvePath('examples/validExamples/subValidExamples');
             exec(`node ${mslg} parse --in ${filePath} --out finalResult -o examples/validExamples/subValidExamples`, (error, stdout, stderr) => {
                 try {
-                    fs.unlinkSync(resolvePath('examples/validExamples/subValidExamples/finalResult_mslg.lg'));
+                    fs.unlinkSync(generatedFilePath + '/finalResult_mslg.lg');
                     assert.equal(stdout.includes('Collated lg file is generated here'), true);
                     done();
                 } catch (err) {

